app/src/main/java/com/example/wizeline/ui/BitsoViewModel.kt

```diff
@@ -0,0 +1,64 @@
+package com.example.wizeline.ui
+
+import androidx.lifecycle.*
+import com.example.wizeline.data.datasource.models.Book
+import com.example.wizeline.data.datasource.models.BookInfoEntity
+import com.example.wizeline.domain.FilterCurrenciesUseCase
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+import com.example.wizeline.data.datasource.models.BidsAndAsksList
+import com.example.wizeline.data.datasource.models.TickerEntity
+import com.example.wizeline.data.repository.CurrencyRepository
+
+class BitsoViewModel(
+    private val repository: CurrencyRepository,
+    private val filterCurrenciesUseCase: FilterCurrenciesUseCase
+    ) : ViewModel(){
+
+    private val _availableBooks = MutableStateFlow(arrayListOf<Book>())
```

Comment:
```suggestion
    private val _availableBooks = MutableStateFlow(emptyListOf())
```

Sugiero que utilices el emptyListOf ya que te devuelve una lista vacia y los datos los vas a manejar a partir de lists y no de arraylists